alerter.cpp

```diff
@@ -24,8 +28,10 @@ void alertInCelcius(float farenheit) {
 }
 
 int main() {
-    alertInCelcius(400.5);
-    alertInCelcius(303.6);
+    alertInCelcius(303.6); // OK
+    assert(alertFailureCount == 0);
+    alertInCelcius(400.5); // NOK

```

Comment:
Tests can be enhanced with following in mind, boundary value, values below and above boundary value.